src/components/Table/Table.tsx

```diff
@@ -2,19 +2,22 @@ import React from "react";
 
 import { Total } from "../Total/Total";
 import { testIds } from "./testIds";
+import { Buttons } from "../Buttons/Buttons";
 
 import table from "./Table.module.css";
 
- type IExpance = {
-    name: string,
-    amount: number
-}
+type IExpance = {
+  name: string;
+  amount: number;
+  id: string;
+};
 
- type IProps = {
-    operationList: IExpance[]
-}
+type IProps = {
+  operationList: IExpance[];
+  removeItem: Function;
```

Comment:
```suggestion
  removeItem: (id: string) => void;
```
`Function` is a bad practice